_exercises/exercise_2.md

```diff
@@ -24,74 +25,34 @@ Webpack is at its core, an _asset bundler_. It specializes in taking all your `.
 
 > Remember that the more "magic" a framework seems to be, and the more it solves, the more important is it that you understand how it works. No frameworks you use in production should feel like a black box you can't explain.
 
-However, Webpack doesn't understand all this without being told what to do. We have to configure it first, and for our app, webpack configuration lives in `config/webpack.config.dev.js` and `config/webpack.config.prod.js`. The former is optimized for developing the app, the latter is optimized for running in production.
+However, Webpack doesn't understand all this without being told what to do. We have to configure it first, and for our app, the `create-react-app` scaffolding tool has provided us with two build configs:
 
-:pencil2: Open `config/webpack.config.prod.js`.
+* A development config optimized for running our app in development mode
+  * Triggered by running the `npm start` command (which you used in the previous exercise)
+* A production config optimized for running in production
+  * Triggered by running the `npm run build` command
 
-This file was created by the folks making `create-react-app`, our scaffolding tool which generated this app for us. The comments explaining the configurations are also written by them, and can be very helpful to understand how it works. In real projects we tend to write this file ourself by hand, because scaffolding tools often support features we won't need or understand, which makes this file seem more bloated, complex and scary than it needs to be. Learning Webpack is a topic worthy of it's own workshop so we won't go into details here, but we'll look at parts of it.
+The development config (`npm start`) starts a special development web server (`webpack-dev-server`), which enables amongst other things the following features:
 
-Webpack is just a "dumb" engine. It's main features or _build tasks_ are separated out into _**plugins**_ and _**loaders**_. Let's go through all of that are used here to de-mystify them a bit, starting with the plugins.
+* Automatic and incremental building_ when files are added or changed without restarting the dev server (so you don't have to run `npm start` all the time)
```

Comment:
en underscore etter _building_. Trenger en underscore til for italic, eller fjern den